runtime/vm/JFRWriter.hpp

```diff
@@ -134,8 +134,13 @@ class VM_JFRWriter {
 	{
 		PORT_ACCESS_FROM_JAVAVM(vm);
 		bool result = true;
+		char *jfrFileName = vm->jfrState.jfrFileName;
```

Comment:
This local should have type `const char *`, making the cast on line 140 unnecessary.